front-end/src/util/__tests__/blockToTime.test.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import blockToTime from '../blockToTime';
```

Comment:
Any reason not to use the same structure as for `formatBnBalance`?
These tests are quite hard to read IMO.
It'd be much easier to read something like:
it ('test blocktotime with and without blocktime set (default to polkadot)')
- test 1
- test 2 
^ both returning the same and `jest.mock for getNetwork` returns 'polkadot'

it('tests that an amount of block lower than 1min returns 0min')<-- actually we should probably return 1min
- test 3

it('tests a time to block equal to 1min returns 1min').

Also to ease readability I'd go with `const SEC = 1000` and then instead of `6000` have `6*SEC`. I wouldn't put the time2 and 3 as variables either. why not for time1 because it's reused but time2 and 3 aren't.